pkg/dynamic/print_config.go

```diff
@@ -91,6 +91,10 @@ func PrintConfig(s interface{}, f io.Writer, style *table.Style, withColor bool,
 						continue
 					}
 					if jtag := tt.Tag.Get("json"); jtag != "" && jtag != "-" {
+						disableConfigPrint := tt.Tag.Get("disableConfigPrint")
```

Comment:
I thought you meant the strategy parameter, for this case, i think we can name it "hidden" or "unprintable"

Reply:
ok @c9s, let's use "unprintable" just because we already have "modifiable"